definitions/divorce/json/CaseEvent/CaseEvent-general-order-nonprod.json

```diff
@@ -0,0 +1,13 @@
+[
+  {
+    "LiveFrom": "27/08/2020",
+    "CaseTypeID": "DIVORCE",
+    "ID": "createGeneralOrder",
+    "Name": "Create general order",
+    "Description": "Create general order",
+    "DisplayOrder": 14,
+    "PreConditionState(s)": "*",
+    "PostConditionState": "*",
+    "SecurityClassification": "Public"
+  }
+]
```

Comment:
Are we sure we dont have any pre or post states for this?

Reply:
yup :)

Reply:
Beautiful